package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
But, I don't understand why you have it at all; you should've been running budo, and don't need npm details or any of these dependencies.